pyaedt/modules/AdvancedPostProcessing.py

```diff
@@ -738,20 +735,18 @@ def animate_fields_from_aedtplt(
 
         Parameters
         ----------
-        plotname : str
+        plot_name : str
             Name of the plot or the name of the object.
         plot_folder : str, optional
             Name of the folder in which the plot resides. The default
             is ``None``.
         variation_variable : str, optional
             Variable to vary. The default is ``"Phase"``.
-        variation_list : list, optional
+        variations : list, optional
             List of variation values with units. The default is
             ``["0deg"]``.
         project_path : str, optional
             Path for the export. The default is ``""`` which export file in working_directory.
```

Comment:
```suggestion
            Path for the export. The default is ``""``, in which case the file is exported
            to the working directory.
```
Avoid unnecessary use of code entities in descriptions.